web/packages/design/src/utils/useResizeObserver.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * Teleport
- * Copyright (C) 2024 Gravitational, Inc.
+ * Copyright (C) 2025 Gravitational, Inc.
```

Comment:
I don't remember we change license date when updating file. INAL though :)

Reply:
Yeah it's not necessary but it doesn't hurt.

Reply:
ah, I just deleted the original file + replaced w/ the new hook, but the diff displays that as updating the cc year